src/payment/strategies/adyenv3/adyenv3-payment-strategy.ts

```diff
@@ -0,0 +1,361 @@
+import { some } from 'lodash';
+
+import { BillingAddress } from '../../../billing';
+import { CheckoutStore, InternalCheckoutSelectors } from '../../../checkout';
+import { getBrowserInfo } from '../../../common/browser-info';
+import { InvalidArgumentError, NotInitializedError, NotInitializedErrorType, RequestError } from '../../../common/error/errors';
+import { OrderActionCreator, OrderRequestBody } from '../../../order';
+import { OrderFinalizationNotRequiredError } from '../../../order/errors';
+import { PaymentArgumentInvalidError, PaymentInvalidFormError, PaymentInvalidFormErrorDetails, PaymentMethodCancelledError } from '../../errors';
+import isVaultedInstrument from '../../is-vaulted-instrument';
+import Payment, { HostedInstrument } from '../../payment';
+import PaymentActionCreator from '../../payment-action-creator';
+import PaymentMethod from '../../payment-method';
+import { PaymentInitializeOptions, PaymentRequestOptions } from '../../payment-request-options';
+import PaymentStrategy from '../payment-strategy';
+
+import { isCardState, AdyenAction, AdyenActionType, AdyenAdditionalAction, AdyenAdditionalActionState, AdyenClient, AdyenComponent, AdyenComponentType, AdyenError, AdyenPaymentMethodType, AdyenPlaceholderData, CardStateErrors } from './adyenv3';
+import AdyenV3PaymentInitializeOptions from './adyenv3-initialize-options';
+import AdyenV3ScriptLoader from './adyenv3-script-loader';
+
+export default class Adyenv3PaymentStrategy implements PaymentStrategy {
+    private _adyenClient?: AdyenClient;
+    private _cardVerificationComponent?: AdyenComponent;
+    private _paymentComponent?: AdyenComponent;
+    private _paymentInitializeOptions?: AdyenV3PaymentInitializeOptions;
+
+    constructor(
+        private _store: CheckoutStore,
+        private _paymentActionCreator: PaymentActionCreator,
+        private _orderActionCreator: OrderActionCreator,
+        private _scriptLoader: AdyenV3ScriptLoader,
+        private _locale: string
+    ) {}
+
+    async initialize(options: PaymentInitializeOptions): Promise<InternalCheckoutSelectors> {
+
+        const { adyenv3 } = options;
+
+        if (!adyenv3) {
+            throw new InvalidArgumentError('Unable to initialize payment because "options.adyenv3" argument is not provided.');
```

Comment:
How will the translation works for this?

Reply:
seems like we don't translate errors all over the checkout-sdk

Reply:
Is this for the end user? or just error in console?